src/pe_source/data/pe_db/db_query.py

```diff
@@ -58,13 +59,27 @@ def get_orgs():
             close(conn)
 
 
+def get_ips(org_uid):
+    """Get IP data."""
+    conn = connect()
+    sql = """SELECT wa.asset as ip_address
+            FROM web_assets wa
+            WHERE wa.organizations_uid = %(org_uid)s
+            and wa.report_on = True
+            """
+    df = pd.read_sql(sql, conn, params={"org_uid": org_uid})
+    ips = list(df["ip_address"].values)
+    conn.close()
+    return ips
+
+
 def get_data_source_uid(source):
     """Get data source uid."""
     conn = connect()
     cur = conn.cursor()
     sql = """SELECT * FROM data_source WHERE name = '{}'"""
     cur.execute(sql.format(source))
```

Comment:
What is the preferred method of string field replacement. I was thinking that it was name = '%s'""" % <parameter_here>. There are multiple instances where this questions applies(sql queries and logging statements).

Reply:
The normal method for string replacement will raise a SQL Injection warning.  Instead, we have to pass the variables inside of the exceute()/execute_values() functions.

Logging statements should all be the method you're describing.

Reply:
@aloftus23 
Thanks for the explanation of the issues that arise from the differences in how we are passing %s into the Postgres queries.